build.gradle

```diff
@@ -316,6 +316,10 @@ dependencies {
     }
     testFixturesImplementation "org.opensearch:common-utils:${version}"
     implementation 'com.github.oshi:oshi-core:6.4.13'
+
+    implementation 'io.github.jbellis:jvector:4.0.0-beta.2-SNAPSHOT'
+    implementation 'org.agrona:agrona:1.20.0'
```

Comment:
Where is this used? I didnt see in the codec

Reply:
should be runtimeOnly

Reply:
It's a transient dependency of jvector, I had to include it because it doesn't resolve automatically. Can make it a runtimeOnly because it's not explicitly used.